EESSI-extend-2023.06-easybuild.eb

```diff
@@ -87,6 +87,16 @@ if (os.getenv("EESSI_CVMFS_INSTALL") ~= nil) then
   end
   eessi_cvmfs_install = true
   easybuild_installpath = os.getenv("EESSI_SOFTWARE_PATH")
+  eessi_accelerator_target = os.getenv("EESSI_ACCELERATOR_TARGET")
```

Comment:
We are expecting people to manually set this, which is probably the safer option (certainly for now), but we could also call out to `lmod_eessi_archdetect_wrapper_accel.sh` and print a warning like:
```
archdetect has discovered an accelerator XXX on your system, if you intend to build for this architecture, you should run:
  EESSI_ACCELERATOR_TARGET=XXX module reload EESSI-extend  
```

Reply:
Would a comment here and an issue be good enough for now?

Reply:
I think this is worth a support ticket (maybe together with the hook request), I would be afraid it gets neglected if it is "just" an issue